awx/main/migrations/0136_scm_track_submodules.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2021-02-02 14:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0135_schedule_sort_fallback_to_id'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='Project',
+            name='scm_track_submodules',
+            field=models.BooleanField(default=False, help_text='Track submodule latest commit on specified branch.'),
+        ),
+        migrations.AddField(
+            model_name='ProjectUpdate',
+            name='scm_track_submodules',
+            field=models.BooleanField(default=False, help_text='Track submodule latest commit on specified branch.'),
+        ),
+        migrations.AlterField(
+            model_name='project',
+            name='scm_track_submodules',
+            field=models.BooleanField(default=False, help_text='Track submodules latest commits on defined branch.'),
```

Comment:
If the addition of `AlterField` operations are merely due to a change in help text, then I'd prefer to condense then into just 2 `AddField` operations before merging.

Reply:
Modified in latest commit to be just 2 AddField